tests/modules/stacked_elmo_lstm_test.py

```diff
@@ -0,0 +1,32 @@
+# pylint: disable=no-self-use,invalid-name
+import numpy
+import torch
+from torch.nn.utils.rnn import pack_padded_sequence
+
+from allennlp.modules.stacked_elmo_lstm import ElmoLstm
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestElmoLstmCell(AllenNlpTestCase):
+    def test_stacked_elmo_lstm(self):
+        input_tensor = torch.autograd.Variable(torch.rand(4, 5, 3))
+        input_tensor[1, 4:, :] = 0.
+        input_tensor[2, 2:, :] = 0.
+        input_tensor[3, 1:, :] = 0.
+        input_tensor = pack_padded_sequence(input_tensor, [5, 4, 2, 1], batch_first=True)
+        lstm = ElmoLstm(num_layers=2,
+                        input_size=3,
+                        hidden_size=5,
+                        cell_size=7,
+                        memory_cell_clip_value=2,
+                        state_projection_clip_value=1)
+        output_sequence, lstm_state = lstm(input_tensor)
```

Comment:
Suggest adding a test here that also passes in an input state. It would have caught the issue above about forward/backward states since the shapes would have been incompatible.

Reply:
One other interesting test to add too and would be easy to incorporate here is to make one of the sequence lengths=0, pass in an initial state, then check that the final state is the same as the initial state.